evmbin/src/display/json.rs

```diff
@@ -18,6 +18,7 @@
 
 use std::collections::HashMap;
 use std::mem;
+use serde::{Serialize, Deserialize};
```

Comment:
shouldn't i be using these imports (i.e. `impl Serialize for Response`). see https://serde.rs/impl-serialize.html

Reply:
good question!